ckanext/ontario_theme/templates/internal/package/snippets/resource_edit_form.html

```diff
@@ -0,0 +1,14 @@
+{% set stage = stage or 1 %}
+{% set include_metadata = include_metadata or false %}
+
+{% extends resource_form_snippet %}
+
+{% block stages %}{% endblock %}
+
+{% block previous_button %}{% endblock %}
+
+{% block again_button %}{% endblock %}
+
+{% block save_button %}
+<button class="ontario-button ontario-button--primary" name="save" type="submit">{% block save_button_text %}Continue{% endblock %}</button>
+{% endblock %}
```

Comment:
Can you format these blocks? Endblocks should have the block name. Please indent the button and the block and text inside the button and block.